docs/topics/db/models.txt

```diff
@@ -1053,6 +1053,21 @@ For example::
         serves_hot_dogs = models.BooleanField(default=False)
         serves_pizza = models.BooleanField(default=False)
 
+Would be equivalent to::
+
+    from django.db import models
+
+    class Place(models.Model):
+        name = models.CharField(max_length=50)
+        address = models.CharField(max_length=80)
+
+    class Restaurant(Place):
```

Comment:
No because we want to maintain that `Restaurant` -> `Place` Python inheritance chain. Making `Restaurant(models.Model)` wouldn't make `restaurant.name` available, it would live under `restaurant.place_ptr.name`.